ios/FluentUI.Demo/FluentUI.Demo/Demos/ColorDemoController.swift

```diff
@@ -117,8 +113,11 @@ class ColorDemoController: UIViewController {
         view.addSubview(stackView)
         view.backgroundColor = Colors.NavigationBar.background
 
-        let window = ColorDemoController.themeWindowType
-        segmentedControl.selectedSegmentIndex = colorProviderThemes.firstIndex(where: { $0.demoColorTheme.windowType == window }) ?? 0
+        if let demoListViewController = navigationController?.viewControllers.first as? DemoListViewController {
```

Comment:
nit: I think you can create a computed property that does this:

`demoListViewController = navigationController?.viewControllers.first as? DemoListViewController`

It's being used in more than one place and could be error prone in case this logic needs to be updated in the future.

```
private var currentDemoListViewController: DemoListViewController? {
    return navigationController?.viewControllers.first as? DemoListViewController
}
```